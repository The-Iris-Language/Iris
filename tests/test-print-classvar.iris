class Animal () {
    public:
        int legs;
        int eyes;
        bool fur;
        string name;
        
        string noise(int volume) {
            return "ruh roh raggy";
        }

    permit: 
        string color;
        bool alive;
}

class Josh () {
    public: 
        int the_goat() {
            return 1;
        }
    
    permit:
        int height;
        int age;
}

class Main () {
    public:
       int univ main () {
            Animal a;
            a.name = "animal name is josh"; 
            Olympus.print(a.name);
            return 0; 
       }
}