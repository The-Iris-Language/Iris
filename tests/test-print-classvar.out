three legs!
animal name is josh
